src/scripts/main.js

```diff
@@ -1,7 +1,39 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const createNotificationElement = () => {
+    const notification = document.createElement('div');
+
+    notification.className = `notification ${type}`;
```

Comment:
You're using a string template to create a class name for the notification. This is a good practice, but it might be better to use a more descriptive name for the `type` parameter, such as `notificationType`.